src/content-script.ts

```diff
@@ -3,12 +3,23 @@ import { logDebug } from './utils'
 
 // https://github.com/pd4d10/json-viewer/issues/28
 // Since Chrome 117
-const el = document.body.lastElementChild
+const elPre = document.querySelector('body > pre')
 
-if (el instanceof HTMLElement && el?.tagName === 'PRE') {
+// Edge Browser
+// https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer
+const elDiv = document.querySelector('body > div[hidden=true]')
+
```

Comment:
Thank you for the contribution!

Could you do a little tweak to remove the redundant code in if/else?

For example,

```js
const el = elDiv ?? elPre

// Then do to the json parse and render
```